src/shared/components/post/post-listing.tsx

```diff
@@ -623,30 +623,6 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
     }
   }
 
-  duplicatesLine() {
```

Comment:
This shouldn't be removed from post-listing, because it shows dupes for the `home` and `community`. 

You'll probably need a *small*, and an expanded version. The expanded would only get shown on the `post.tsx` page. I think the best way here, is to add a prop to `post-listing`, like

```typescript

type ShowDupesType = "Small" | "Expanded" | "None"
...

PostlistingProps {
...
showDupes: ShowDupesType`
```

Reply:
Its really useful, especially when someone makes a lot of crossposts, and you don't want to see them all duped on your front page. Don't worry about it for now tho, I'll do a PR to this one with some of my suggestions in a few hours.